core/src/main/java/org/apache/iceberg/ManifestWriter.java

```diff
@@ -85,11 +85,15 @@ void addEntry(ManifestEntry<F> entry) {
         deletedRows += entry.file().recordCount();
         break;
     }
+
     stats.update(entry.file().partition());
-    if (entry.sequenceNumber() != null
-        && (minSequenceNumber == null || entry.sequenceNumber() < minSequenceNumber)) {
-      this.minSequenceNumber = entry.sequenceNumber();
+
+    if (entry.isLive()
```

Comment:
Attention! I think there is no behavior change as `sequenceNumber()` was returning null for DELETEd entries.
See [here](https://github.com/apache/iceberg/pull/5913/files#r987078407) for details.